app/Helpers/VideoHelper.php

```diff
@@ -14,7 +14,7 @@ function generateThumbnail($videoPath)
         ]);
 
         $video = $ffmpeg->open($videoPath);
-        $frame = $video->frame(TimeCode::fromSeconds(10));
+        $frame = $video->frame(TimeCode::fromSeconds(2));
```

Comment:
Good Reducing video seconds to 2 makes it reliable due to short videos less than 10 seconds would not be generating Thumbnail.